docs/source/users/index.md

```diff
@@ -5,6 +5,23 @@ Welcome to the user documentation for Jupyter AI.
 If you are interested in contributing to Jupyter AI, 
 please see our {doc}`contributor's guide </contributors/index>`.
 
+## Prerequisites
+
+You will need a supported version of Python up to and including 3.10. Python 3.11 is **not supported** due
+to incompatibility with the [ray](https://pypi.org/project/ray/) library that we use.
+
+If you use `conda`, you can install Python 3.10 in your environment by running:
+
+```
+conda install python=3.10
+```
+
+You will also need to have a currently-maintained version of JupyterLab installed. If you use `conda`, you can install JupyterLab in your environment by running:
+
+```
+conda install jupyterlab
```

Comment:
Can we just add jupyterlab as a dependency to the jupyter_ai package so users don't have to think about it?